### PR TITLE
fields ui modifications

### PR DESCRIPTION
**Pull request by stevenslack**
* adds a hover state to field group handlers
* collapsible toggler added to collapsible fields
* aria-expanded attribute added to the collapsible toggler

### CODE REVIEW

File: php/class-fieldmanager-field.php [hunk 1]
```diff
@@ -939,7 +939,7 @@ public function get_remove_handle() {
 	 * @return string
 	 */
 	public function get_collapse_handle() {
-		return sprintf( '<div class="handlediv" title="%s"><br /></div>', esc_attr__( 'Click to toggle', 'fieldmanager' ) );
```
Comment (mboynes) [inline php/class-fieldmanager-field.php]: Funny, so there was HTML markup for something, but it wasn't set to render anywhere?

File: css/fieldmanager.css [hunk 2]
```diff
@@ -32,29 +32,62 @@
...
+.fm-group-inner, .fm-group-label-wrapper {
+	border: solid 1px #dfdfdf;
+	-webkit-box-shadow: 0 1px 1px rgba(0,0,0,.04);
+	box-shadow: 0 1px 1px rgba(0,0,0,.04);
 }
 
 .fm-group-inner {
-	padding: 5px;
+	padding: 10px;
+    border-top: none;
```
Comment (mboynes) [inline css/fieldmanager.css]: Looks like this got indented using spaces for some reason

File: css/fieldmanager.css [hunk 3]
```diff
@@ -32,29 +32,62 @@
...
+	display: block;
+}
+
+.fm-group-label-wrapper.fmjs-collapsible-handle:hover .toggle-indicator {
+	color: #777;
+}
+
+.fm-group .toggle-indicator {
+	color: #a0a5aa;
+    margin-left: 5px;
```
Comment (mboynes) [inline css/fieldmanager.css]: indented with spaces

File: css/fieldmanager.css [hunk 4]
```diff
@@ -32,29 +32,62 @@
...
+}
+
+.fm-group .toggle-indicator:before {
+	vertical-align: middle;
+}
+
+.fm-item .fmjs-collapsible-handle.closed .toggle-indicator:before {
+    content: "\f140";
```
Comment (mboynes) [inline css/fieldmanager.css]: indented with spaces

File: php/class-fieldmanager-field.php (hunk 1 above)
Comment (stevenslack) [inline php/class-fieldmanager-field.php]: I couldn't find it anywhere.

File: css/fieldmanager.css [hunk 5]
```diff
@@ -32,16 +32,19 @@
 
 /* Groups */
 .fm-group {
-	border: solid 1px #dfdfdf;
 	margin-bottom: 5px;
 	background-color: #fff;
+}
+
+.fm-group-inner, .fm-group-label-wrapper {
```
Comment (NerdcoreSteve) [inline css/fieldmanager.css]: should be a newline after comma

File: css/fieldmanager.css [hunk 6]
```diff
@@ -32,16 +32,19 @@
...
 	box-shadow: 0 1px 1px rgba(0,0,0,.04);
 }
 
 .fm-group-inner {
 	padding: 10px;
-	border-top: 1px solid #dfdfdf;
+	border-top: none;
 }
 
 .fm-group h4, .fm-group div.fm-group-label-wrapper {
```
Comment (NerdcoreSteve) [inline css/fieldmanager.css]: newline after comma

**Comment by NerdcoreSteve** [comment]
Looks good to me :smiley_cat: :+1:

**Pull request merged**